docs/docsite/rst/roadmap/COLLECTIONS_2_10.rst

```diff
@@ -0,0 +1,39 @@
+====================
+Ansible project 2.10
+====================
+
+This release schedule includes dates for the `ansible <https://pypi.org/project/ansible/>`_ package, with a few dates for the `ansible-base <https://pypi.org/project/ansible-base/>` package as well. All dates are subject to change, please see :ref:`base_roadmap_2_10` for the most recent updates on ansible-base.
+
+.. contents::
+   :local:
+
+Release Schedule
+----------------
+
+.. note:: Dates subject to change.
+.. note:: We plan to post weekly alpha releases to the `PyPI ansible project <https://pypi.org/project/ansible/>`_ for testing.
+
+- 2020-06-23: ansible-2.10 alpha freeze.
```

Comment:
I wonder if it would be worth marking the past items as done, so it's cleared on achieved vs aims

Reply:
if we do that, we'd have to update this page every time a date passes, which seems like more work than it's worth